### PR TITLE
Improve PEtab support

### PR DESCRIPTION
**Pull request by yannikschaelte**
* Fix wrong import of log distributions (wrong arguments, wrong scale before). Principle: Prior returns parameters on prior scale, therefore model has to rescale inputs.
* Add tests for prior generation
* Discard that overly complicated allowing to estimate fixed parameters too
* Now, pyABC should support all priors defined in PEtab.
* Note: Currently sampling is always performed on prior scale, which should be statistically correct , but ignores the PEtab idea of defining a separate parameter scale. How that is to be interpreted in terms of transformation is yet TBD.

### CODE REVIEW

File: test/petab/test_petab.py [hunk 1]
```diff
@@ -1,13 +1,173 @@
...
+    # -- MEAN AND VARIANCE -- #
+
+    # sample mean and variance
+    mean = np.mean(samples)
+    var = np.var(samples)
+
+    # ground truth mean and variance
+    if prior_type in [C.UNIFORM, C.PARAMETER_SCALE_UNIFORM]:
+        mean_th = var1 + (var2 - var1) / 2
+        var_th = (var2 - var1)**2 / 12
```
Comment (dweindl) [inline test/petab/test_petab.py:90]: Where does the `12` come from?

**Review by dweindl — APPROVED** [review]
Not feeling overly confident, but looks good to me.

File: test/petab/test_petab.py (hunk 1 above)
Comment (yannikschaelte) [reply test/petab/test_petab.py:90]: that's the variance of  (a,b) uniforma variable https://en.wikipedia.org/wiki/Continuous_uniform_distribution

Comment (dweindl) [reply test/petab/test_petab.py:90]: I guess one should know, thanks :see_no_evil:

**Review by LeonardSchmiester — APPROVED** [review]
Same as for Daniel regarding confidence level. But looks fine from what I can tell.

**Pull request merged**